lib/msal-browser/src/request/SsoSilentRequest.ts

```diff
@@ -25,5 +25,8 @@ import { AuthorizationUrlRequest } from "@azure/msal-common";
  * - domainHint                 - Provides a hint about the tenant or domain that the user should use to sign in. The value of the domain hint is a registered domain for the tenant.
  * - extraQueryParameters       - String to string map of custom query parameters.
  * - nonce                      - A value included in the request that is returned in the id token. A randomly generated unique value is typically used to mitigate replay attacks.
+ * - forceRefresh               - Force a network request, instead of leveraging cached credentials. Defaults to false.
```

Comment:
An argument could be made that this should default to `true`, which is the current behavior. I personally think the semantics of `forceRefresh` should align with the other libraries, regardless of the current behavior of `ssoSilent`, as most apps will welcome the improved default performance of `ssoSilent`.

Reply:
I'm fine with defaulting to false and just mentioning it in the changelog as "potentially breaking"

Reply:
Thought about this over the weekend, and I think I will default to true, to maintain existing behavior, which apps may be relying on. Will document this and denote this behavior will likely change in future major version.